tests/functional/config_files/environment.py

```diff
@@ -39,7 +39,7 @@ def get_env_file(variable_name: str) -> str:
     'client_id': get_env('CLIENT_ID'),
     'client_secret': get_env('CLIENT_SECRET'),
     'nhs_login_private_key': get_env('ID_TOKEN_NHS_LOGIN_PRIVATE_KEY_ABSOLUTE_PATH'),
-    'jwt_private_key': get_env('JWT_PRIVATE_KEY_ABSOLUTE_PATH'),
+    'jwt_private_key': get_env('APPLICATION_RESTRICTED_SIGNING_KEY_PATH'),
```

Comment:
Could you please pull latest master and check this, I feel Paz's change may have already fixed this